<?xml version="1.0"?>
<launch>
  <group ns="rtabmap">

    <!--node pkg="nodelet" type="nodelet" name="odom_sync" args="standalone rtabmap_ros/data_odom_sync" output="screen">
      <remap from="rgb/image_in" to="/camera/rgb/image_rect_color"/>
      <remap from="depth/image_in" to="/camera/depth_registered/image_raw"/>
      <remap from="rgb/camera_info_in" to="/camera/rgb/camera_info"/>
      <remap from="odom_in" to="/odom"/>
    </node-->

      <!-- Use RGBD synchronization -->

    <node pkg="nodelet" type="nodelet" name="rgbd_sync" args="standalone rtabmap_ros/rgbd_sync" output="screen">

      <remap from="rgb/image" to="/camera/rgb/image_raw"/>
      <remap from="depth/image" to="/camera/depth_registered/image_raw"/>
      <remap from="rgb/camera_info" to="/camera/rgb/camera_info"/>
      <remap from="rgbd_image" to="rgbd_image"/>

      <param name="approx_sync" value="true"/>

     </node>

   </group>
</launch>
